pkg/controller/node/node_controller_test.go

```diff
@@ -213,6 +214,60 @@ func (f *fixture) expectPatchNodeAction(node *corev1.Node, patch []byte) {
 	f.kubeactions = append(f.kubeactions, core.NewPatchAction(schema.GroupVersionResource{Resource: "nodes"}, node.Namespace, node.Name, types.MergePatchType, patch))
 }
 
+func TestGetNodesForPool(t *testing.T) {
```

Comment:
1 liner comment for this?

Reply:
This is testing the `getNodesForPool` function - following how `Test...` in golang are written

Reply:
ah sorry, still drinking coffee